server/routes/api/messages.js

```diff
@@ -9,11 +9,11 @@ router.post("/", async (req, res, next) => {
       return res.sendStatus(401);
     }
     const senderId = req.user.id;
-    const { recipientId, text, conversationId, sender } = req.body;
+    const { recipientId, text, conversationId, sender, read } = req.body;
```

Comment:
Isn't all new message unread by default? I think we should remove `read` property from request body.

Reply:
Agreed, `read` removed